payment_payzen/models/payment_transaction.py

```diff
@@ -69,6 +70,16 @@ def _get_specific_rendering_values(self, processing_values):
             'vads_cust_phone': self.partner_phone and self.partner_phone[0:31] or '',
         })
 
+        if self.sale_order_ids.recurring_payment_provider_id:
+            if len(self.sale_order_ids) > 1:
+                raise ValidationError(_("Unsupported recurring payment transaction (%s) with multiple sale orders: %s") % (self, self.sale_order_ids))
```

Comment:
Translations should be with `,` instead of `%`
```suggestion
                raise ValidationError(_("Unsupported recurring payment transaction (%s) with multiple sale orders: %s", self, self.sale_order_ids))
```
Same for the ones below

Reply:
AFAIK using `%` is allowed since 15.0 IIRC (as long as the _() object is receiving it and not the str inside).
It's also featured in the examples from the docs: https://www.odoo.com/documentation/17.0/developer/howtos/translations.html#blocks